subsys/bluetooth/controller/ll_sw/nordic/lll/lll_central_iso.c

```diff
@@ -707,7 +689,16 @@ static void isr_rx(void *param)
 	cis_lll = param;
 
 	/* No Rx */
-	if (!trx_done) {
+	if (!trx_done ||
+#if defined(CONFIG_TEST_FT_CEN_SKIP_SUBEVENTS)
+	    ((((cis_lll->event_count % 3U) < CONFIG_TEST_FT_CEN_SKIP_EVENTS_COUNT) &&
+	      ((se_curr > cis_lll->nse) || (se_curr <= 2U))) ||
+	     (((cis_lll->event_count % 3U) < (CONFIG_TEST_FT_CEN_SKIP_EVENTS_COUNT + 1U)) &&
+	      ((se_curr > cis_lll->nse) || (se_curr <= 1U)))) ||
```

Comment:
It is hard to read this and understand what it does. Can you add documentation or move this to a function or something to help the reader?

Reply:
Added comment.